examples/evm/mappingchallenge.py

```diff
@@ -0,0 +1,61 @@
+from manticore.ethereum import ManticoreEVM, Detector
+################ Script #######################
+
+m = ManticoreEVM()
+m.verbosity(3)
+#And now make the contract account to analyze
+#https://capturetheether.com/challenges/math/mapping/
+source_code = '''
+pragma solidity ^0.4.21;
+
+contract MappingChallenge {
+    bool public isComplete;
+    uint256[] map;
+
+    function set(uint256 key, uint256 value) public payable {
+        // Expand dynamic array as needed
+        if (map.length <= key) {
+            map.length = key + 1;
+        }
+
+        map[key] = value;
+    }
+}
+'''
+print "Source code:\n",source_code
```

Comment:
This isn't py3 compatible, although maybe we don't care for the samples just yet.